prepare_database.go

```diff
@@ -42,6 +43,8 @@ func defaultInitDatabase(binaryExtractLocation, username, password, locale strin
 		return fmt.Errorf("unable to init database using: %s", postgresInitDbProcess.String())
 	}
 
+	os.Remove(passwordFile)
```

Comment:
Let's check this return error and propagate it if necessary with a useful message as above?

May want to add coverage for this one, just check that it's been removed perhaps in one of the existing tests?